### DISCUSSION TITLE
Wrong content redirection issue for collaborator and content creator

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-Knowlg/Community#206] [Bugs]
Hello, we're using the Sunbird 5.0 version.
Whenever a collaborator adds multiple pieces of content to the same course unit, the material is incorrectly misdirected for both the collaborator and the content author.

A collaborator adding a unit to the course and adding multiple pieces of content within that unit will cause the unit to redirect to another piece of content when the collaborator clicks on it.
And this is taking place for both content creators and collaborators. It functions well for the remaining users.
For reference, kindly watch the attached videos.
Kindly provide a fix or workaround for the same.

https://github.com/Sunbird-Knowlg/Community/assets/109504832/8b305b73-a6ff-4318-8be4-3e6dd8f863a0

**Comment by pallakartheekreddy** [comment]
@sachinb80 Details provided above not clear. Can you please help me understand what is you exact issue? and the video you shared also not clear.

**Reply by sachinb80** [reply]
@pallakartheekreddy Ok i'll share the new video

**Reply by sachinb80** [reply]
@pallakartheekreddy If needed, can we connect on this issue for a better understanding? This is a bit of a confusing issue if i show it to you directly, you will understand it better. Please let me know a suitable time. so that i can schedule a meeting

**Reply by sachinb80** [reply]
@pallakartheekreddy here is the new video for the issue. Please let me know if you face any issues understanding this

https://github.com/Sunbird-Knowlg/Community/assets/109504832/00a1c378-147d-44f9-8f64-061c232bdbc1

**Reply by pallakartheekreddy** [reply]
@sachinb80 Can we get into a call to check this issue.

**Reply by sachinb80** [reply]
@pallakartheekreddy  Sure. Please let me know a suitable time. I will schedule a call

**Reply by pallakartheekreddy** [reply]
@sachinb80 lets connect 12:45PM

**Reply by sachinb80** [reply]
Sure. i have sent you the invite.

**Reply by pallakartheekreddy** [reply]
@sachinb80 looks like this is sunbird portal issue. Please create a thread in below forum.
https://github.com/orgs/Sunbird-Ed/discussions

@vinukumar-vs @rajeevsathish FYR.

**Reply by sachinb80** [reply]
Sure

**Reply by pallakartheekreddy** [reply]
@sachinb80 Please raise this issue in the ED discussion forum. For now, closing this discussion.
